x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -228,15 +228,58 @@ describe('when on the list page', () => {
 
           firstPolicyID = hostListData[0].metadata.Endpoint.policy.applied.id;
 
-          [HostStatus.ERROR, HostStatus.ONLINE, HostStatus.OFFLINE, HostStatus.UNENROLLING].forEach(
-            (status, index) => {
-              hostListData[index] = {
-                metadata: hostListData[index].metadata,
-                host_status: status,
-                query_strategy_version: queryStrategyVersion,
-              };
-            }
-          );
+          // add ability to change (immutable) policy
+          type DeepMutable<T> = { -readonly [P in keyof T]: DeepMutable<T[P]> };
```

Comment:
is `DeepMutable` the inverse of `Immutable`? :)
I thought maybe it woud be `Mutable<>`
Also - be careful with making thing mutable. With deep structures, that can cause some trouble when we start to get side affects. Especially if data is not fully "reset" beteween each test.

Reply:
Deep because it's a nested immutability in this field.

a new object is created in `makePolicy` function each call, and that is what's mutated.